nautobot/core/api/views.py

```diff
@@ -519,6 +519,8 @@ def get(self, request):
             if version:
                 if isinstance(version, tuple):
                     version = ".".join(str(n) for n in version)
+                else:
+                    version = str(version)
```

Comment:
At least one of the updated dependencies was returning a version that was a non-JSON-serializable Python object, so I had to add this.